flink/v1.12/flink/src/main/java/com/netease/arctic/flink/InternalCatalogBuilder.java

```diff
@@ -66,7 +74,39 @@ public InternalCatalogBuilder metastoreUrl(String metastoreUrl) {
   }
 
   public InternalCatalogBuilder properties(Map<String, String> properties) {
-    this.properties = properties;
+    Map<String, String> finalProperties = new HashMap<>();
```

Comment:
Why adding logic here if CatalogLoader had been implemented to support config these properties?

Reply:
> 
Because we support "auth.kerberos.krb.path" and "auth.kerberos.keytab.path", we should read this config files and encode in flink client.